src/instrumentation.ts

```diff
@@ -54,13 +54,13 @@ export class AzureFunctionsInstrumentation extends InstrumentationBase {
             azFunc.app.hook.preInvocation((context) => {
```

Comment:
Can you explain the diff here? 
I don't see any change other than the fallback to `otelContext.active()` or am I missing something?
And in this case, the fallback will never get executed since propagation.extract always returns a Context object – never undefined. See link for reference
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/context/api-propagators.md#extract

So `extractedContext || otelContext.active()` will always yield to extractedContext.

Reply:
This always returns a Context object, even if nothing could be extracted. So:

`const extractedContext = propagation.extract(...);`
extractedContext will never be undefined or null, and will always be a valid Context instance (even if identical to the input context).

The issue was older code was returning a null value.

Reply:
I don't understand, if extract() always returns a Context object how can it return null in the old code?
We might need to fix that or else if the extract returns null in the new code too it will default to otelContext.active() which might not be the trace propogation that is expected.